front/.yarnrc.yml

```diff
@@ -1 +1,3 @@
+nodeLinker: node-modules
+
```

Comment:
Aren't we supposed to use the yarn cache?

Reply:
fixed

Reply:
Nope, we need to keep it that way or `yarn` won't install all dependencies and build fails.
Try to build it locally without this property, you'll need to explicitly install some peer/sub dependencies.

And this still leverages the cache on installation.

Reply:
cf https://yarnpkg.com/features/linkers